src/NUnitFramework/nunit.framework.legacy.tests/nunit.framework.legacy.tests.csproj

```diff
@@ -16,7 +16,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="NUnit3TestAdapter" />
-    <PackageReference Include="NUnit.Analyzers" />
+    <PackageReference Include="NUnit.Analyzers">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Autogenerating by the VS tooling. I opted not to revert it as it may continue to keep happening on each update, but open to thoughts or ideas

Reply:
Because we use `Directory.Packages.props` there is no need to update the `.csproj` at all.
VS tooling doesn't seem to recognize that.
I normally just update the version number manually.

Reply:
Happy to do that, thanks! I'll give this another go tomorrow evening again.

We should probably squash merge this when the time comes to keep the history clean

Reply:
Updated. I've adjusted this and squashed on my end so that the PR itself is simpler.
Thanks for the suggestion!